configs/nuxtjs.json

```diff
@@ -2,19 +2,37 @@
   "index_name": "nuxtjs",
   "start_urls": [
     {
-      "url": "https://nuxtjs.org/(?P<lang>.*?)/",
+      "url": "https://nuxtjs.org/fr/",
       "variables": {
         "lang": [
```

Comment:
I'd suggest using `lang` instead of `language` so that you don't have any changes to do on the nuxtjs doc!

Reply:
thanks :heart:

Reply:
@danielroe https://github.com/algolia/docsearch-configs/pull/4619/commits/62ca71b9a9af23bde72e6e5f6cc457f1658acd74 should do it! 

For future updates, our scraper also supports `docsearch:` meta tags: https://docsearch.algolia.com/docs/required-configuration/#introduce-global-information-as-meta-tags

Or selecting the `lang` from the `html` element ([like this](https://github.com/algolia/docsearch-configs/blob/master/configs/vuejs-v3.json#L22-L26)) but it seems to be broken on https://nuxtjs.org/fr/ right now, not sure if you're aware of it!

Reply:
Amazing - so that sounds like it should be possible to omit the list of start_urls + extra_attributes entirely and simply search via language code instead (for example, `facetFilters: ["lang:en"]`)?

Reply:
Exactly, you'd only have `https://nuxtjs.org/` in the `start_urls` and the i18n would be handled by our scraper either by the meta tags or the `lang` selector!

To avoid any changes in your documentation, [this selector](https://github.com/algolia/docsearch-configs/blob/master/configs/vuejs-v3.json#L22) can be called `language` so you'd keep the actual `facetFilters: ["language:en"]`